server.js

```diff
@@ -0,0 +1,85 @@
+var mongo = require('mongodb').MongoClient;
+var port = process.env.PORT || '8080';
+var mongoConnectionPath = 'mongodb://127.0.0.1/chatDB' || process.env.MONGOLAB_URI;
```

Comment:
in this line precess.env will never be used, as string 'mongodb://127.0.0.1/chatDB' will always be true.